File: Views/src/app/_components/administrative/local/financial/components/checking-account/dto/checking-account-dto.ts
import { CardDto } from "../../card/dto/card-dto";

export class CheckingAccountDto {
  id:number;
  institution:string;
  holder:string;
  agency:number;
  account:number;
  pix:string;
  typeaccount:string;
  cards: CardDto[];
  description:string;
}

File: Services/Dto/CheckingAccountDto.cs

using System.Collections.Generic;

namespace Services.Dto
{
    public class CheckingAccountDto
    {
        public int Id { get; set; }
        public string Institution { get; set; }
        public string Holder { get; set; }
        public string Agency { get; set; }
        public string Manager { get; set; }
        public string Account { get; set; }
        public string Pix { get; set; }
        public string Typeaccount { get; set; }
        public List<CardDto> Cards {get; set;}
        public string Description { get; set; }

    }
}